README.md

```diff
@@ -39,5 +48,5 @@ If you have any questions about Submitting PR's, Opening Issues, aws-lc API usag
 Otherwise, if you think you might have found a security impacting issue, please instead follow [our Security Notification Process.](#security-issue-notifications)
```

Comment:
Is this Gitter chatroom now public? If so, may be comment on the issue?

Reply:
Yup, it's public now.